syntax/expr_binary_test.go

```diff
@@ -19,13 +19,19 @@ func TestRelationCall(t *testing.T) {
 
 func TestOpsAddArrow(t *testing.T) {
 	t.Parallel()
+	// Tuples
```

Comment:
Please create separate test `func`s for different test cases instead of mixing them in one.